### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by zarva6596**

### CODE REVIEW

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let i = 0;
+
+  return (a) => {
+    return (b) => {
+      if (i < 3 || i / 2 === Math.floor(i / 2)) {
```
Comment (zarichnyi) [inline src/makeRobotAccountant.js:13]: наглядніша перевірка була б на залишок остачі
i % 2 === 0

**Review by zarichnyi — APPROVED** [review]

File: src/makeRobotAccountant.js (hunk 1 above)
Comment (vkryvytskyy) [reply src/makeRobotAccountant.js:13]: use this hint, just like doen't change your code)

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -6,7 +6,21 @@
...
+        i++;
+
+        return a + b;
+      } else {
+        i++;
```
Comment (vkryvytskyy) [inline src/makeRobotAccountant.js:18]: you make i++ in both cases, maybe move it outside if..else?

**Review by vkryvytskyi — CHANGES_REQUESTED** [review]

File: src/makeRobotAccountant.js [hunk 3]
```diff
@@ -6,7 +6,19 @@
...
 
 function makeRobotAccountant() {
-  // write code here
+  let i = 0;
+
+  return (a) => {
+    return (b) => {
+      i++;
+
+      if (i - 1 < 3 || (i - 1) % 2 === 0) {
```
Comment (vkryvytskyy) [inline src/makeRobotAccountant.js:15]: i believe you can write solution without using `i - 1`

**Review by vkryvytskyi — CHANGES_REQUESTED** [review]

File: src/makeRobotAccountant.js [hunk 4]
```diff
@@ -12,7 +12,7 @@ function makeRobotAccountant() {
     return (b) => {
       i++;
 
-      if (i - 1 < 3 || (i - 1) % 2 === 0) {
+      if (i <= 3 || (i) % 2 !== 0) {
```
Comment (vkryvytskyy) [inline src/makeRobotAccountant.js:15]:
```suggestion
      if (i <= 3 || i % 2 !== 0) {
```

**Review by vkryvytskyi — APPROVED** [review]

**Pull request closed without merging**